docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.2"
+services:
+  ckanextgeodatagov:
+    build: 
+      context: ckan-docker
+    env_file:
+      - .env
+    links:
+      - db
+      - solr
+      - redis
+      - datapusher
+    ports:
+      - "0.0.0.0:${CKAN_PORT}:5000"
+    volumes:
+      - ckan_storage:/var/lib/ckan
+      - .:/srv/app/src_extensions/geodatagov
+  db:
+    image: ckan/postgresql
+    expose:
+      - "5432"
+  solr:
+    image: ckan/solr
+    restart: always
+    expose:
+      - "8983"
+  datapusher:
```

Comment:
ckanext-geodatagov is only installed on Catalog where we're not using datapusher. Any reason to include it in the development setup?